### PR TITLE
RCM-98: Alphabetical order in List of drugs displayed on UI on Drug Order tab

### PR DESCRIPTION
**Pull request by jberchmas**
Adding alphabetical order in List of drugs displayed on Drug Order tab to help user to find drug very quickly

### CODE REVIEW

File: omod/src/main/java/org/openmrs/web/controller/analysis/CohortBuilderController.java [hunk 1]
```diff
@@ -168,7 +169,19 @@ public int compare(Concept left, Concept right) {
 			model.put("encounterTypes", Context.getEncounterService().getAllEncounterTypes());
 			model.put("locations", Context.getLocationService().getAllLocations());
 			model.put("forms", Context.getFormService().getAllForms());
-			model.put("drugs", Context.getConceptService().getAllDrugs());
+			List<Drug> drugs=Context.getConceptService().getAllDrugs();
+			Collections.sort(drugs, new Comparator<Drug>() {
+				public int compare(Drug d1, Drug d2) {
+					if(d1.getName().compareToIgnoreCase(d2.getName())<0){
```
Comment (dkayiwa) [inline omod/src/main/java/org/openmrs/web/controller/analysis/CohortBuilderController.java]: What happens when name is null?

Comment (lluismf) [inline omod/src/main/java/org/openmrs/web/controller/analysis/CohortBuilderController.java]: Null Pointer Exception 

You can use StringUtils.compareIgnoreCase (it's null safe)

**Comment by dkayiwa** [comment]
@jberchmas did you see the above comments?

**Comment by dkayiwa** [comment]
@jberchmas how is this going?

**Comment by dkayiwa** [comment]
@jberchmas am closing this for now. I can always reopen whenever you get some time to finish it up. :)

**Pull request closed without merging**